### ISSUE TITLE
Cryptic error when project description is too long (and field too short)

### ISSUE THREAD

**Issue by jiho** [oceanomics/ecotaxa_dev#429]
```
Error: <class 'sqlalchemy.exc.DataError'> 
Description: (raised as a result of Query-invoked autoflush; consider using a session.no_autoflush block
if this flush is occurring prematurely) (psycopg2.errors.StringDataRightTruncation)
ERREUR:  valeur trop longue pour le type character varying(50)
 [SQL: 'UPDATE projects SET projtype=%(projtype)s WHERE projects.projid = %(projects_projid)s']
[parameters: {'projtype': 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, ... (148 characters
truncated) ... illum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident,
sunt in culpa qui officia deserunt mollit anim id est laborum.', 'projects_projid': 3070}] (Background
on this error at: http://sqlalche.me/e/9h9h) 
```

50 characters is short and there should be a check in the UI, not an SQL achelmy error.

**Comment by jiho** [comment]
And the difference between project description and Comments is not clear.

**Comment by grololo06** [comment]
Error fixed for 2.4.5 while moving "Edit Project" to back-end
